css/footer.css

```diff
@@ -0,0 +1,24 @@
+/* Footer */
+
+footer {
```

Comment:
Maybe avoid styling elements directly. I know in this case it makes sense to style the footer element directly but still, it's not good practice.

Reply:
Yes, I won't be reusing the footer element anywhere else, so it won't be an issue here. 
Thanks for suggesting, I'll keep that in mind!